backend/models/sessionsModel.js

```diff
@@ -0,0 +1,9 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const sessionSchema = new Schema({
+  cookieId: { type: String, required: true, unique: true },
+  createdAt: { type: Date, expires: 30, default: Date.now }
+});
+
+module.exports = mongoose.model('Session', sessionSchema);
```

Comment:
How are you able to know which user a session cookie belongs to?

Reply:
ahhh, we have not gotten up to session cookies yet :x

Reply:
@mjhult we're going to add session cookies in another PR.